coverage:
  all_nodes: 0
  covered_nodes: 0
mutation_score: 25.0
mutations:
- exception_traceback: null
  killer: null
  module: &id001 !!python/module:app ''
  mutations:
  - lineno: 22
    operator: AOR
  number: 1
  status: survived
  tests_run: 16
  time: 0.051523447036743164
- exception_traceback: null
  killer: null
  module: *id001
  mutations:
  - lineno: 99
    operator: AOR
  number: 2
  status: survived
  tests_run: 16
  time: 0.06316113471984863
- exception_traceback: null
  killer: null
  module: *id001
  mutations:
  - lineno: 177
    operator: AOR
  number: 3
  status: survived
  tests_run: 16
  time: 0.0722811222076416
- exception_traceback: null
  killer: null
  module: *id001
  mutations:
  - lineno: 180
    operator: AOR
  number: 4
  status: survived
  tests_run: 16
  time: 0.05981087684631348
- exception_traceback: null
  killer: null
  module: *id001
  mutations:
  - lineno: 183
    operator: AOR
  number: 5
  status: survived
  tests_run: 16
  time: 0.05612897872924805
- exception_traceback: "def test_start_equals_goal():\n        \"\"\"\n        Test\
    \ the trivial case where the start and goal are the same.\n        The expected\
    \ result is an immediate success with a path containing only the start.\n    \
    \    \"\"\"\n        grid = [\n            [0, 0],\n            [0, 0]\n     \
    \   ]\n        start = Node(0, 0)\n        goal = Node(0, 0)\n        # Set the\
    \ start node's id and parent id.\n        start.id = 0\n        start.pid = 0\n\
    \        astar = AStar(grid)\n        found, path = astar.plan(start, goal)\n\
    >       assert found is True, \"Path should be found when start equals goal\"\n\
    E       AssertionError: Path should be found when start equals goal\n\ntest/test_app.py:30:\
    \ AssertionError"
  killer: test/test_app.py::test_start_equals_goal
  module: *id001
  mutations:
  - lineno: 83
    operator: COI
  number: 6
  status: killed
  tests_run: 1
  time: 0.07367396354675293
- exception_traceback: "def test_simple_path():\n        \"\"\"\n        Test a simple\
    \ 5x5 grid with no obstacles.\n        The start is at the top\u2010left and the\
    \ goal at the bottom\u2010right.\n        The algorithm should find a path.\n\
    \        \"\"\"\n        n = 5\n        grid = [[0 for _ in range(n)] for _ in\
    \ range(n)]\n        start = Node(0, 0)\n        goal = Node(4, 4)\n        start.id\
    \ = 0\n        start.pid = 0\n        astar = AStar(grid)\n        found, path\
    \ = astar.plan(start, goal)\n        assert found is True, \"A path should be\
    \ found in an open grid\"\n        # According to our conversion, the returned\
    \ path is from goal to start.\n>       assert compare_coordinates(path[0], goal),\
    \ \"The first node in the path should be the goal\"\nE       IndexError: list\
    \ index out of range\n\ntest/test_app.py:51: IndexError"
  killer: test/test_app.py::test_simple_path
  module: *id001
  mutations:
  - lineno: 138
    operator: COI
  number: 7
  status: killed
  tests_run: 2
  time: 0.06957387924194336
- exception_traceback: null
  killer: null
  module: *id001
  mutations:
  - lineno: 61
    operator: IOD
  number: 8
  status: survived
  tests_run: 16
  time: 0.05815005302429199
number_of_tests: 16
targets:
- src
tests:
- name: test_app
  target: null
  time: 0.12127113342285156
- name: conftest
  target: null
  time: 0.030898571014404297
time_stats:
  create_mutant_module: 0.013257741928100586
  create_target_ast: 0.00479435920715332
  mutate_module: 1.8456599712371826
  run_tests_with_mutant: 0.5077033042907715
total_time: 2.002647638320923
